tests/shutdown/shutdown.pm

```diff
@@ -31,9 +31,12 @@ sub run {
     if (!sle_version_at_least('12-SP2') && check_var('VIRTIO_CONSOLE', 1)) {
         add_serial_console('hvc0');
     }
+    # Shutdown network only on qemu backend
+    if (ckeck_var('BACKEND', 'qemu')) {
+        systemctl 'stop network.service';
+        systemctl 'stop wickedd.service';
+    }
     # Clean dhcp ids from hdd image
-    systemctl 'stop network.service';
-    systemctl 'stop wickedd.service';
     assert_script_run('ls /var/lib/wicked/');
```

Comment:
isn't SLED using NM and not wicked?

Reply:
yes, same as openSUSE on a real laptop profile

Reply:
You are right, I will add more checks.

Reply:
@coolo @okurz I added more conditions ` if (!(get_var('FLAVOR', '') =~ /^Desktop/ || check_var('SLE_PRODUCT', 'sled')) || is_sle) {`